isda_daycounters/thirtyE360ISDA.py

```diff
@@ -0,0 +1,40 @@
+"""
+ISDA 30E/360 (ISDA) day count convention.
+=====
+
+Implemented according to 2006 ISDA definition:
+https://www.isda.org/2008/12/22/30-360-day-count-conventions/
+"""
+
+import calendar
+
+name = 'thirtyE/360 ISDA'
+
+def day_count(start_date, end_date, is_end_date_on_termination=False):
+    """Returns number of days between start_date and end_date, using ThirtyE/360 ISDA convention"""
+
+    def is_last_day_of_feb(curr_date):
```

Comment:
Maybe move this out and prefix it with an underscore to indicate privacy, i.e. `_is_last_day_of_feb`.

Reply:
Makes sense. Will make the change. Thanks